shell/platform/fuchsia/dart_runner/BUILD.gn

```diff
@@ -221,7 +220,6 @@ template("jit_runner_package") {
     binary = "dart_jit${product_suffix}_runner"
 
     cmx_file = rebase_path("meta/dart_jit${product_suffix}_runner.cmx")
-    cml_file = rebase_path("meta/dart_jit${product_suffix}_runner.cml")
```

Comment:
Since chase designed the build rules to include both a cmx file and cml file for v2 migrated components, we also have the option of only using the v1 path instead of reverting this entire PR.

Reply:
The V1 pathway regressed because the fidl_compatibility_tests in-tree make use of this path / dart_runner

See: https://source.corp.google.com/fuchsia/src/tests/fidl/compatibility/